paddle/fluid/API.spec

```diff
@@ -1,6 +1,7 @@
 paddle.fluid.Program.__init__ ArgSpec(args=['self'], varargs=None, keywords=None, defaults=None)
 paddle.fluid.Program.block ArgSpec(args=['self', 'index'], varargs=None, keywords=None, defaults=None)
 paddle.fluid.Program.clone ArgSpec(args=['self', 'for_test'], varargs=None, keywords=None, defaults=(False,))
+paddle.fluid.Program.convert_to_test_program ArgSpec(args=['self'], varargs=None, keywords=None, defaults=None)
```

Comment:
This API has some potential issues:
train program and test program can be quite different. It's unlikely that there is a way to automatically convert train program to test program. So the API should be called "prune_train_ops_and_reader_ops". Obviously, the name looks strange. So I'm not sure we need such an API.